src/pkg/cli/client/byoc/aws/byoc.go

```diff
@@ -487,8 +523,13 @@ func (b *ByocAws) runCdCommand(ctx context.Context, cmd cdCommand) (ecs.TaskArn,
 	}
 	env["DEFANG_MODE"] = strings.ToLower(cmd.mode.String())
 	if cmd.dockerHubUsername != "" && cmd.dockerHubAccessToken != "" {
-		env["CI_REGISTRY_USER"] = cmd.dockerHubUsername
-		env["CI_REGISTRY_PASSWORD"] = cmd.dockerHubAccessToken // FIXME: use config for this
+		arn, err := b.putDockerHubSecret(ctx, cmd.project, cmd.dockerHubUsername, cmd.dockerHubAccessToken)
+		if err != nil {
+			term.Warnf("Could not store Docker Hub credentials in Secrets Manager, images from dockerhub may be throttled during build: %v", err)
+		} else {
+			env["CI_REGISTRY_CREDENTIALS_ARN"] = arn
```

Comment:
isn't this always "the same" ARN?

Reply:
Secrets manager secrets ARN has a 6 character random suffix added to it. It will always be the same, but we cannot predict the suffix.